community/modules/compute/gke-job-template/variables.tf

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+variable "name" {
+  description = "The name of the job."
+  type        = string
+  default     = "my-job"
+}
+
+variable "node_count" {
+  description = "How many nodes the job should run in parallel."
+  type        = number
+  default     = 1
+}
+
+variable "command" {
+  description = "A list of strings that will be joined to create the job command."
```

Comment:
nit, maybe rephrase this using cloud build as [inspiration](https://cloud.google.com/build/docs/configuring-builds/create-basic-configuration):
"(...)  takes a list of arguments and passes them to the <container> referenced by <image>"
Isn't this more accurate?

Reply:
It is a little different than the cloud batch case because it is the binary and the arguments. 

I have updated with wording from k8s documentation.